cli.js

```diff
@@ -31,7 +31,7 @@ function runScripts(scripts, files) {
 	});
 }
 
-var cli = meow([
+const cli = meow([
 	'Usage',
```

Comment:
The `meow` help should use a template literal. See: https://github.com/sindresorhus/trash-cli/blob/e6dee38aa399758f6d2a66fa0cb56b052f327cfa/cli.js#L7